www/src/components/Menu.tsx

```diff
@@ -144,16 +148,40 @@ interface IManuQueryComponents {
   all: Array<IComponentNavItem>
 }
 
+const foundationLinks = [
+  'Colors', 'Elevation', 'Typography', 'Layout', 'Spacing', 'Icons', 'CSS-Utilities', 'Responsive',
+];
+
 function Menu() {
+  const {
+    settings,
+    handleSettingsChange,
+  } = useContext(SettingsContext);
   const { components } = useStaticQuery(menuQuery);
-  const { categories, all }: IManuQueryComponents = components;
+  const { categories }: IManuQueryComponents = components;
 
   return (
-    <Container size="xl" className="py-5">
-      <Row className="mb-5">
-        <Col sm={3}>
-          <h2 className="mb-4 h3">Guides</h2>
-          <ul className="list-unstyled small mb-5">
+    <div className="pgn-doc__menu">
+      <h2 className="pgn-doc__menu-title">Theme</h2>
+      <div className="pgn-doc__menu-btn--group">
```

Comment:
We have [ButtonGroup](https://paragon-openedx.netlify.app/components/button/button-group/) component for such cases, should probably use it instead

Reply:
It`s good idea, thanks